indico/modules/events/abstracts/controllers/abstract_list.py

```diff
@@ -95,6 +95,13 @@ class RHAbstractList(DisplayAbstractListMixin, RHAbstractListBase):
     template = 'management/abstract_list.html'
     view_class = WPManageAbstracts
 
+    def _process(self):
+        if self.list_generator.static_link_used:
+            return redirect(self.list_generator.get_list_url())
+        kwargs = self.list_generator.get_list_kwargs()
+        kwargs['track_session_map'] = {track.id: track.default_session_id for track in self.event.tracks}
```

Comment:
Not sure if this is the best way of passing `track_session_map` to the template, I just didn't want to add it in `DisplayAbstractListMixin` since it's probably being used by other classes.

Reply:
Yes, that's fine. We do the same e.g. in `PaperAssignmentListGenerator`

Reply:
Actually, looking at it again, I'd rather override `get_list_kwargs` and do it in there (that's what we do in the class I mentioned above)

Reply:
Why not override `get_list_kwargs` in `AbstractListGeneratorManagement`?

Reply:
Or hm, it's not really related to the list generator. So maybe adding it there wouldn't be the best choice.

Why not override `_render_template` and add it to the kwargs in there, and then call super?

Reply:
Good point, updated.

Reply:
Whoops I missed your 2nd comment

Reply:
Updated